### DISCUSSION TITLE
Эмбиент с Bigcity

### DISCUSSION THREAD

**Discussion by Emberenius** [octothorp-team/public-dobrograd#667] [Предложения]
Предлагаю вернуть старый эмбиент карты. Было очень атмосферно находиться на сервере при малом онлайне и слышать вдалеке какие-то звуки живого города. Просто очень скучаю по старой атмосфере еще зимней карты 2020

**Comment by FlinMan0** [comment]
Да.

**Comment by LENFYS** [comment]
За

**Comment by travka4000** [comment]
я не олд так-что не буду голосовать
